config/locales/en/main.yml

```diff
@@ -605,6 +605,7 @@ en:
           Treat as text: Accept options for this option set as if they were text entries"
       qing_group:
         group_item_name: "You can enter $QuestionCode to include the value of a question (for ODK Collect only).\n\nFor example, suppose you are making a repeat group where the code for the first question is 'HouseholdID' and you enter\n\n&nbsp;&nbsp;&nbsp;Household: $HouseholdID \n\n You create one group and answer '123' for 'HouseholdID.' Then you create a second group and answer '789' for 'HouseholdID.' The first group item name will be 'Household: 123.' The second group item name will be 'Household: 789.'"
+        repeat_count: "You can enter $QuestionCode to a previous question to set a fixed amount of times a repeat group is repeated. Integers and calculations are not supported at this time."
```

Comment:
tested with enketo and it's unsupported, let's add this: "ODK Collect only; integers and calculations are not supported at this time."

Reply:
(whole string will also need to be updated if we make it a dropdown)